src/reducers/transitionProperties.ts

```diff
@@ -0,0 +1,57 @@
+import { HIDE_TRANSITION_PROPERTIES, SHOW_TRANSITION_PROPERTIES, SET_JSON_MESSAGE_STRING, SET_CURRENT_ID, SET_PRIVATE_TOPIC, GlobalTransitionPropertiesAction } from "actions/transitionProperties";
+import sessionXCSpy from "utils/sessionXCSpy";
+import { Reducer } from "redux";
+
+export interface TransitionPropertiesState {
+    active: boolean;
+    stateMachine: string;
+    messageType: string;
+    jsonMessageString: string;
+    id: string;
+    privateTopic: string;
+};
+
+let initialState: TransitionPropertiesState = {
+    active: false,
+    stateMachine: undefined,
```

Comment:
@maurelio1234 I don't know what the best practice is. Use undefined or null as default value?